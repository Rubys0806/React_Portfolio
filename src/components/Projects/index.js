import React from 'react';
import { capitalizeFirstLetter } from '../../utils/helpers';

<section id="projects" class="projects">
<h2>Projects:</h2>
<p>
    <a href=" https://github.com/hlachaal/food-truck-finder"><div class="box"> Food Truck Finder </div></a>
   <a href=" https://github.com/defhook/pass"><div class="box"> Pass </div></a>
   <a href=" https://github.com/frivera14/Travel-Planner"><div class="box"> Travel Planner </div></a>
</p>
</section>

export default Projects;
