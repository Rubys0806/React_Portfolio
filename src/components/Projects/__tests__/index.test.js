import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Projects from '..'



<section id="projects" class="projects">
<h2>Projects:</h2>
<p>
    <a href=" https://github.com/hlachaal/food-truck-finder"><div class="box"> Food Truck Finder </div></a>
   <a href=" https://github.com/defhook/pass"><div class="box"> Pass </div></a>
   <a href=" https://github.com/frivera14/Travel-Planner"><div class="box"> Travel Planner </div></a>
</p>
</section>
