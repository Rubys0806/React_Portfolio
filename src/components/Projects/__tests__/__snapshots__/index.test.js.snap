// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component matches snapshot 1`] = `
<DocumentFragment>

        <section id="projects" class="projects">
            <h2>Projects:</h2>
            <p>
                <a href=" https://github.com/hlachaal/food-truck-finder"><div class="box"> Food Truck Finder </div></a>
               <a href=" https://github.com/defhook/pass"><div class="box"> Pass </div></a>
               <a href=" https://github.com/frivera14/Travel-Planner"><div class="box"> Travel Planner </div></a>
            </p>
        </section>
        
</DocumentFragment>
`;
